apps/trackers/tests/test_file_offer.py

```diff
@@ -40,7 +40,11 @@ class TestTrackerSuggestions:
                 Affect.AffectResolution.WONTFIX,
                 False,
             ),
-            (Affect.AffectAffectedness.NEW, Affect.AffectResolution.WONTFIX, False),
+            (
+                Affect.AffectAffectedness.NEW,
+                Affect.AffectResolution.DEFER,
+                False,
```

Comment:
It makes sense to test the DEFER resolution but we should just add it and not replace the WONTFIX case.